src/Sentry/Internal/ModuleExtensions.cs

```diff
@@ -0,0 +1,19 @@
+namespace Sentry.Internal;
+
+internal static class ModuleExtensions
+{
+    /// <summary>
+    /// The Module.Name for Modules that are embedded in SingleFileApps will be null
+    /// or &lt;Unknown&gt;, in that case we can use Module.ScopeName instead
+    /// </summary>
+    /// <param name="module">A Module instance</param>
+    /// <returns>module.Name, if this is available. module.ScopeName otherwise</returns>
+    public static string? GetNameOrScopeName(this Module module) =>
+#if TRIMMABLE
+        module?.ScopeName;
+#else
+        (module?.Name is null || module.Name.Equals("<Unknown>"))
```

Comment:
if module is null, isn't `module.Name` going to to throw null exception?

Reply:
If `module?.Name is null` then the expression evaluates to `true`, so `module.Name` never gets evaluated.